bcda/cclf/importer.go

```diff
@@ -33,9 +24,13 @@ type cclf8Importer struct {
 }
 
 // validates that cclf8Importer implements the interface
-var _ importer = &cclf8Importer{}
+// var _ importer = &cclf8Importer{}
 
-func (cclfImporter *cclf8Importer) do(ctx context.Context, tx *sql.Tx, fileID uint, b []byte) error {
+func (cclfImporter *cclf8Importer) do(ctx context.Context, tx *sql.Tx, data interface{}) error {
```

Comment:
Sorry, I didn't realize the interface goes away once we removed the `importCCLF()` method. Because there's no importer interface, your old way of supplying the models.CCLFBeneficiary makes more sense.